CredentialProvider.Microsoft/CredentialProviders/Vsts/VstsCredentialProvider.cs

```diff
@@ -55,82 +51,67 @@ public override async Task<bool> CanProvideCredentialsAsync(Uri uri)
 
         public override async Task<GetAuthenticationCredentialsResponse> HandleRequestAsync(GetAuthenticationCredentialsRequest request, CancellationToken cancellationToken)
         {
-            // If this is a retry, let's try without sending a retry request to the underlying bearerTokenProvider
```

Comment:
It doesn't look like we're doing this anymore (the line below was also removed: "var responseWithNoRetry = await HandleRequestAsync(GetNonRetryRequest(request), cancellationToken);").

I'm not sure I understand why we were doing this recursive call with IsRetry: false before. Was this change on purpose?